doc/cortx-aws-k8s-installation.md

```diff
@@ -260,6 +269,14 @@ mv ./cortx-k8s/k8_cortx_cloud/solution.yaml ./cortx-k8s/k8_cortx_cloud/solution.
 # Update list of disks and list of nodes in the solutions.yaml file.
 ./cortx-k8s/k8_cortx_cloud/generate-cvg-yaml.sh --nodes nodes.txt --devices devices.txt --cvgs 2 --data 2 --solution ./cortx-k8s/k8_cortx_cloud/solution.yaml.orig  --datasize $DiskSize --metadatasize $DiskSize > ./cortx-k8s/k8_cortx_cloud/solution.yaml
 ```
+
+The memory allocation and resource utilization for the nodes can also be configured in `solution.yaml`.
+The c5.2xlarge instances used in this guide have too little memory for the default `large` configuration,
+so change to `small`.
+```
+yq -i '.solution.common.setup_size = "small"' cortx-k8s/k8_cortx_cloud/solution.yaml
+```
+
 #### 3.2.2.1 Update the images tag.
 
 - If you are using your own generated cortx-all image, update the image tag on the solution.yaml file:
```

Comment:
The list of images (below) is not consistent with the current deployment, nor do the version numbers look what I would expect.

The v0.0.22 version of solution.yaml looks like this:
```
  images:
    cortxcontrolprov: ghcr.io/seagate/cortx-all:2.0.0-642-custom-ci
    cortxcontrol: ghcr.io/seagate/cortx-all:2.0.0-642-custom-ci
    cortxdataprov: ghcr.io/seagate/cortx-all:2.0.0-642-custom-ci
    cortxdata: ghcr.io/seagate/cortx-all:2.0.0-642-custom-ci
    cortxserver: ghcr.io/seagate/cortx-all:2.0.0-642-custom-ci
    cortxha: ghcr.io/seagate/cortx-all:2.0.0-642-custom-ci
    cortxclient: ghcr.io/seagate/cortx-all:2.0.0-642-custom-ci
```